src/amber/cli/templates/app/config/environments/development.yml.ecr

```diff
@@ -18,9 +16,11 @@ database_url: postgres://postgres:@localhost:5432/<%= database_name_base %>_deve
 <%when "sqlite" -%>
 database_url: sqlite3:./db/<%= database_name_base %>_development.db
 <%end -%>
-session:
-  key: amber.session
-  store: :signed_cookie
-  expires: 0
-secrets:
-  description: Store your development secrets credentials and settings here.
+session: {
+  key: amber.session,
+  store: signed_cookie,
+  expires: 0,
+}
+secrets: {
```

Comment:
Why does this have curly brackets? Shouldn't it just be a tab to turn into has with yaml?

Reply:
In my tests with yaml it seems to work just fine without them...

```yaml
secrets:
  name: "isaac"
  age: 34
  hash: true
```